app/src/main/index.js

```diff
@@ -95,53 +115,90 @@ ipcMain.on('open-cropper-window', (event, size) => {
+          height: event.height
+        };
+        mainWindow.webContents.send('cropper-window-new-size', size);
+        settings.setSync('cropperWindow.size', size);
+
+        const position = {
+          x: event.x,
+          y: event.y
+        };
+        settings.setSync('cropperWindow.position', position);
+
+        selectorBounds = event;
+      });
+      selectorWindow.on('blur', () => {
+        if (!recording) {
+          selectorWindow.kill();
+          selectorWindow = null;
+        }
       });
     } else {
-      setCropperWindowOnBlur();
-    }
+      cropperWindow = new BrowserWindow({
+        width: width + cropperWindowBuffer,
+        height: height + cropperWindowBuffer,
+        frame: false,
+        transparent: true,
+        resizable: true,
+        shadow: false,
+        x,
+        y
+      });
+      cropperWindow.loadURL(`file://${__dirname}/../renderer/views/cropper.html`);
+      cropperWindow.setIgnoreMouseEvents(false); // TODO this should be false by default
+
+      if (isDev) {
+        cropperWindow.openDevTools({mode: 'detach'});
+        cropperWindow.webContents.on('devtools-opened', () => {
+          setCropperWindowOnBlur();
+        });
+      } else {
+        setCropperWindowOnBlur();
+      }
 
-    cropperWindow.on('closed', () => {
-      cropperWindow = undefined;
-      mainWindow.webContents.send('cropper-window-closed');
-    });
+      cropperWindow.on('closed', () => {
+        cropperWindow = undefined;
+        mainWindow.webContents.send('cropper-window-closed');
+      });
 
-    cropperWindow.on('resize', () => {
-      const size = {};
-      [size.width, size.height] = cropperWindow.getSize();
-      mainWindow.webContents.send('cropper-window-new-size', size);
-      settings.setSync('cropperWindow.size', size);
-    });
+      cropperWindow.on('resize', () => {
+        const size = {};
+        [size.width, size.height] = cropperWindow.getSize();
+        mainWindow.webContents.send('cropper-window-new-size', size);
+        settings.setSync('cropperWindow.size', size);
+      });
 
-    cropperWindow.on('move', () => {
-      const position = {};
-      [position.x, position.y] = cropperWindow.getPosition();
-      settings.setSync('cropperWindow.position', position);
-    });
+      cropperWindow.on('move', () => {
+        const position = {};
+        [position.x, position.y] = cropperWindow.getPosition();
+        settings.setSync('cropperWindow.position', position);
+      });
+    }
   }
 });
 
 ipcMain.on('close-cropper-window', () => {
-  if (cropperWindow && !recording) {
+  if ((cropperWindow || selectorWindow) && !recording) {
     mainWindow.setAlwaysOnTop(false); // TODO send a PR to `menubar`
     menubar.setOption('alwaysOnTop', false);
-    cropperWindow.close(); // TODO: cropperWindow.hide()
+    if (cropperWindow) {
+      cropperWindow.close(); // TODO: cropperWindow.hide()
+    } else {
+      selectorWindow.kill();
+      selectorWindow = null;
```

Comment:
Also not sure this is needed. I also accidentally inconsistently set it to null, where ahead I set it to undefined.